contents/handbook/people/hiring-process/index.mdx

```diff
@@ -298,6 +296,17 @@ As a rule, all interviews at PostHog are conducted in English. Whilst this might
 
 If you are paired with an interviewee who speaks your native language, just politely acknowledge this and let them know all interviews are conducted in English. We also require these calls to be conducted as a video call, so a working webcam is necessary.
 
+### How interviews are scored
+
+Scoring Scale (1– 4)
+
+-   1: Strong No = This candidate is clearly not a fit for PostHog now or in the future
+-   2: No = Not a fit now (maybe in the future)
+-   3: Yes = This is a solid hire
+-   4: Strong Yes = This is an exceptional person we need to hire. (we might go to extra lengths to hire them)
+
+> A good rule of thumb when deciding whether not to progress at any stage: if the candidate doesn't get a _definite yes_ then assume it's a _no_. It's almost never worth putting through someone who is a 'maybe'! We provide lots of information about PostHog to enable candidates to put their best application forward.
```

Comment:
Is a "definite yes" the same thing as a "strong yes"? Or is it like a "yes yes score 3 or 4"?